test/spec/features/modeling/DeleteConnectionSpec.js

```diff
@@ -67,6 +87,19 @@ describe('features/modeling - remove connection', function() {
   }));
 
 
+  it('should remove Outgoing', inject(function(modeling, elementRegistry) {
```

Comment:
Should we add a test case to verify batch deletion works ("no double deletion happens")?

Reply:
Should be in `DeleteElementsSpec` though.